draft-ietf-mmusic-sdp-bundle-negotiation.xml

```diff
@@ -1125,27 +1125,28 @@ SDP Answer
          <t>
              However, some implementations of may not include this identification-tag
              in their RTP and RTCP traffic when using the BUNDLE mechanism,
-             and instead use a payload type based mechanism for demuxing.
+             and instead use a payload type based mechanism to associate RTP streams with SDP m= lines.
              In this situation, each "m=" line MUST use unique payload type values,
              in order for the payload type to be a reliable indicator of the relevant
              "m=" line for the RTP stream.
-             Note that when using payload type based demuxing, an SSRC
-             will be mapped to an "m=" line by the first packet with that
-             SSRC, and the mapping will not be changed even if the same
-             SSRC is received with a different payload type.  In other
+             Note that when using the payload type to associate RTP streams 
+             with m= lines an RTP stream, identified by SSRC, will be mapped 

```

Comment:
The confusion this causes is that implementers do not think m=lines are identified by SSRC. They point out there is no requirement for the SDP to have SSRC values in it at all.